helm_deploy/cmd-api/templates/cronjob-shift-changes-refresh.yaml

```diff
@@ -4,11 +4,11 @@ kind: CronJob
 metadata:
   name: "refresh-shift-changes"
 spec:
-  schedule: "0 19 * * *"
+  schedule: "0 6,12,18 * * *"
```

Comment:
Does this mean that we're only refreshing the tokens 3 times a day at 6am, 12pm, 6pm? I thought we wanted to do it more frequently than that to keep the notifications on the app more up to date

Reply:
it takes 2 hours to run 😂

Reply:
I'm hoping the extra memory below will help, but we're limited by the query I think.
this was hidden before because they were running 12 different calls which took 20 mins each..

Reply:
we could look at splitting the calls into threads in the code though?

Reply:
Works for now 🤷🏽‍♀️

Might be worth adding a ticket to the board for this to be reviewed by the new team to see if the notification lag is an issue for users